docs/book/src/clusterctl/commands/upgrade.md

```diff
@@ -28,9 +28,9 @@ control-plane-kubeadm   capi-kubeadm-control-plane-system   ControlPlaneProvider
 cluster-api             capi-system                         CoreProvider             v0.4.0           v1.0.0
 infrastructure-docker   capd-system                         InfrastructureProvider   v0.4.0           v1.0.0
 
-
+```
```

Comment:
I believe the excluded text is actually a part of the output.

Reply:
Yup I'll remove those